cpp/CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ else()
 endif()
 
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/cmake)
-find_package(ClangFormat 8.0.0 EXACT REQUIRED)
+find_package(ClangFormat 8.0 EXACT REQUIRED)
```

Comment:
Did you face any issue with the version string here?

Reply:
I installed clang-format, but have version 8.0.1, and cmake wouldn't accept it. Relaxing the version bounds here fixed the problem for me. Obviously we can change it back, and I can install 8.0.0, but I thought others might have a similar problem.

Reply:
If you install the `libclang` conda package from rapidsai channel, that should resolve this issue. We recommend sticking to a specific version of clang-format in order to avoid differences arising in formatting between its versions itself.

Reply:
It's a good point. I hadn't seen the clang-format availability from the `rapids` conda channel. I've reverted the change and pushed.